gpii/node_modules/flowManager/src/UntrustedFlowManager.js

```diff
@@ -76,34 +83,34 @@ fluid.defaults("gpii.request.flowManager.untrusted.stateChangeHandler", {
         onSettings: null
     },
     listeners: {
-        onUserToken: [
-            {
-                priority: "first",
-                listener: "gpii.request.flowManager.setUserToken",
-                args: ["{that}", "{arguments}.0"]
-            },
-            "{that}.getDeviceContext"
-        ],
+        "onUserToken.setUserToken": {
+            listener: "gpii.flowManager.setUserToken",
+            args: ["{that}", "{arguments}.0"]
+        },
+        "onUserToken.getDeviceContext": {
+            func: "{that}.getDeviceContext",
+            priority: "after:setUserToken"
+        },
         onDeviceContext: "{that}.getSettings",
         onSettings: "{that}.startLifecycle"
     }
 });
 
-fluid.defaults("kettle.requests.request.handler.untrustedUserLogin", {
-    gradeNames: ["fluid.littleComponent", "gpii.request.flowManager.untrusted.stateChangeHandler"],
+fluid.defaults("gpii.flowManager.untrusted.userLogin.handler", { // TODO: Duplicate other than parent grade
```

Comment:
This grade is quite similar with the other grade ```gpii.flowManager.userLogin.handler``` in ```UserLogonStateChange.js``` except one base grade. Can we refactor the common parts into a grade for reuse?